docs/resources/Message.md

```diff
@@ -510,6 +517,23 @@ Additionally, the combined sum of characters in all `title`, `description`, `fie
 
 Embeds are deduplicated by URL.  If a message contains multiple embeds with the same URL, only the first is shown.
 
+#### Embed Fields by Embed Type
+
+Certain embed types are used to power special UIs. These embeds use [fields](#DOCS_RESOURCES_MESSAGE/embed-object-embed-field-structure) to include additional data in key-value pairs. Below is a reference of possible embed fields for each of the following embed types. 
+
+###### Poll Result Embed Fields
+
+| Embed Field Name              | Embed Field Value Description                              |
```

Comment:
```suggestion
| Field                        | Description                                                |
```

Reply:
just standardizing with the tables above

Reply:
> just standardizing with the tables above

Tbh I prefer the way it ~~currently is~~ was, since it's **not** just like a normal object...